### PR TITLE
adding ansible tests

### PR DESCRIPTION
**Pull request by ianzhang366**

### CODE REVIEW

File: kubeconfigs/hub [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
...c2ZuMnk1UVBoZEJPVVhHbko1L2p2NDZ4T292YU9DOApXRWRmMkQvQXJaaEhvdFFkdWpmbnBYRkk3Nk1samZNV2pqUFIzeG02YlRFSU9iTGEvTjNlWFkzSWdpUngxR2ZvCmdHOUJCdmxKTjNaNUtaelBHeTBxRmNvbEpJcm9mYlBJSHNrMmpPTE9ubVVzT2FON0RKZlZIUmNDZ1lFQTJzSm0KWWpzWEZHdmpiaXREQ2J6L1RpVlpic2JCN1h4WmsvQmlMTzl1Ymp1Tm1HZkRMSHJBOGNobzZjdWFUV1RxL1pMYwp3bVBlMjFHalF4UWNyMks3aUcyTGVVdEhaMURKV3J5RE1ESDF4MUtobEtSN1A4azRFYTFFQ2VORmd3dUtkQU1pCnQrNHFUcjZ5eng1eHBhT1paYUo4ZFdldGtXdGd6V0dGQXl0aGlGc0NnWUVBNHpGU000Ym8yV1o0R2U5TlZIcm8KcWRUYXJBNWY1MFB2SzBZb3h0bkpNYUFVNVR3eXcvbUh3YXdldjMwQjJMRTU3Y2ZuWnNFcGJZbWxybEp2bW13UgpIUmtKOVR5aTFiZEVpalMxTENOWnkydllkdEQ2a28zcnpXVW1DeEFNK3dkMU9nNUw3SWFNbGdOa0pQcXROQll3CjI2TUJseTJzQkk5ZmZSTmt1MlY2T0UwQ2dZRUEwK2p1L3djWW8ycWpwR3Y2QTZ2MmNML1RjM0RNR1A2VlU2N1AKbTVIUW1xM0x5eS93cTl5NW5UMnpDNDkwaUxaU3dKS01rUzNuaXZMNVFSaHhkU0FpWTlVaC83eUJqYjUvWHdzdgoxZmIvakRjcVdQQkFpQmltM05ESUVQLzhyU1gzMTh2VnpQREVRUVdQOXYwWndBZkxwOTJQekxkRE5wUWE4TGtmCi9XTC9sdk1DZ1lCanZ3VGRxaXpNVTR1aWFHY0pWMHlFVHRObWtMR3hXWUlxSTJNc2E2MktmT2pkL2d4TkZlOTUKWjM1bVhqK3NqTVBrMElub2orZmZDWHZZRW9IMyt5djg2ekVmZVFLRThzY2NnVlI2WFYvR3RLcTFPbUNtckVUegovQldkWU5aV0kzeGFMZDUvSmpRSEQwclFMdWlzdDJYTGwvRFVsdlVQZ0w0N285aWI4WGl4TkE9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo=
```
Comment (pezhang) [inline kubeconfigs/hub:19]: @ianzhang366 Seems we have to override this file (kubeconfigs/hub) when running with different cluster. Is there any specific reason to change the content of this example like this?  Thanks.

File: testdata/testcases/ansible-e2e.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "test_id": "ansible-002",
+  "desc": "post ansible job should be created(note a secret with tower access should be created a the subscription namespace(ansible-post) before this test case is run)",
+  "urls": ["https://raw.githubusercontent.com/ianzhang366/acm-applifecycle-samples/e2e/git-ops/ansible/appsub-with-post/appsub/00-namespace.yaml",
+    "https://raw.githubusercontent.com/ianzhang366/acm-applifecycle-samples/e2e/git-ops/ansible/appsub-with-post/appsub/01-placement-rule.yaml",
+    "https://raw.githubusercontent.com/ianzhang366/acm-applifecycle-samples/e2e/git-ops/ansible/appsub-with-post/appsub/02-channel.yaml",
+    "https://raw.githubusercontent.com/ianzhang366/acm-applifecycle-samples/e2e/git-ops/ansible/appsub-with-post/appsub/03-subscription.yaml"
+  ],
+  "target_cluster": "hub"
+}
+]
```
Comment (pezhang) [inline testdata/testcases/ansible-e2e.json:22]: @ianzhang366 Do we need to write the expectation file of this 2 test cases? Thanks.

File: kubeconfigs/hub (hunk 1 above)
Comment (ianzhang366) [reply kubeconfigs/hub:19]: nope, I tried to add this to .gitignore but somehow it's not ignoring this one...

File: testdata/testcases/ansible-e2e.json (hunk 2 above)
Comment (ianzhang366) [reply testdata/testcases/ansible-e2e.json:22]: yup, I will do another PR to add them... since the ansible env is not stable yet

Comment (pezhang) [reply testdata/testcases/ansible-e2e.json:22]: Make sense.

**Comment by pezhang** [comment]
Looks good to me.

Ack.

**Review by pezhang — APPROVED** [review]

**Pull request merged**